src/ubx.h

```diff
@@ -671,6 +673,9 @@ class GPSDriverUBX : public GPSHelper
 	const Interface		_interface;
 	uint32_t _survey_in_acc_limit;
 	uint32_t _survey_in_min_dur;
+
+	// ublox Dynamic platform model default 7: airborne with <2g acceleration
+	uint8_t _dynModel{7};
```

Comment:
It was to match the field straight out of ublox documentation, but I'll change it.